### PR TITLE
NFDIV-2461 Add Welsh Translation

### PR DESCRIPTION
**Pull request by BWeeks-JusticeGovUK**
https://tools.hmcts.net/jira/browse/NFDIV-2461

### CODE REVIEW

File: src/main/steps/applicant1/hub-page/sole/content.ts [hunk 1]
```diff
@@ -378,18 +378,18 @@ const cy: typeof en = (
     link: config.get('govukUrls.moneyAndProperty'),
   },
   finalOrderRequested: {
-    line1: 'You have applied for a ‘final order’. Your application will be checked by court staff.',
-    line2: `If there are no other applications that need to be completed then your ${
-      isDivorce ? 'marriage' : 'civil partnership'
-    } will be legally ended.`,
+    line1: "Rydych wedi gwneud cais am 'orchymyn terfynol'.  Bydd eich cais yn cael ei wirio gan staff y llys.",
```
Comment (velimd) [inline src/main/steps/applicant1/hub-page/sole/content.ts:381]: multiple spaces after the period.

File: src/main/steps/applicant1/hub-page/sole/content.ts [hunk 2]
```diff
@@ -378,18 +378,18 @@ const cy: typeof en = (
...
+    line2: `Os nad oes unrhyw geisiadau eraill y mae angen eu cwblhau yna bydd eich ${
+      isDivorce ? 'priodas' : 'partneriaeth sifil'
+    } yn dod i ben yn gyfreithiol.`,
     line3: `${
       dayjs().isAfter(userCase.dateFinalOrderNoLongerEligible)
-        ? `You will receive an email by ${getFormattedDate(
+        ? `Byddwch yn cael e-bost erbyn  ${getFormattedDate(
```
Comment (velimd) [inline src/main/steps/applicant1/hub-page/sole/content.ts:387]: mutliple spaces before the variable

File: src/main/steps/respondent/hub-page/content.ts [hunk 3]
```diff
@@ -319,17 +319,17 @@ const cy: typeof en = ({ isDivorce, partner, userCase, contactEmail }: CommonCon
     buttonLink: `${RESPONDENT}${FINALISING_YOUR_APPLICATION}`,
   },
   finalOrderRequested: {
-    line1: `Your ${partner} has applied for a ‘final order’. The application will be checked by court staff. If there are no other applications that need to be completed then your ${
-      isDivorce ? 'divorce will be finalised' : 'civil partnership will be legally ended'
+    line1: `Mae eich ${partner} wedi gwneud cais am 'orchymyn terfynol'.  Bydd y cais yn cael ei wirio gan staff y llys. Os nad oes unrhyw geisiadau eraill y mae angen eu cwblhau yna bydd eich ${
```
Comment (velimd) [inline src/main/steps/respondent/hub-page/content.ts:322]: same as above

File: src/main/steps/respondent/hub-page/content.ts [hunk 4]
```diff
@@ -319,17 +319,17 @@ const cy: typeof en = ({ isDivorce, partner, userCase, contactEmail }: CommonCon
...
     line2: `${
       dayjs().isAfter(userCase.dateFinalOrderNoLongerEligible)
-        ? `You will receive an email by ${getFormattedDate(
+        ? `Byddwch yn cael e-bost erbyn ${getFormattedDate(
             dayjs(userCase.dateFinalOrderSubmitted).add(config.get('dates.finalOrderSubmittedOffsetDays'), 'day'),
             SupportedLanguages.Cy
-          )}`
-        : 'You should receive an email within 2 working days,'
-    } confirming whether the final order has been granted.`,
+          )} yn cadarnhau `
```
Comment (danielbe11) [inline src/main/steps/respondent/hub-page/content.ts:330]:
```suggestion
          )} yn cadarnhau`
```

File: src/main/steps/respondent/hub-page/content.ts [hunk 5]
```diff
@@ -319,17 +319,17 @@ const cy: typeof en = ({ isDivorce, partner, userCase, contactEmail }: CommonCon
...
+        : 'Dylech gael e-bost o fewn 2 ddiwrnod gwaith, gan gadarnhau '
```
Comment (danielbe11) [inline src/main/steps/respondent/hub-page/content.ts:331]:
```suggestion
        : 'Dylech gael e-bost o fewn 2 ddiwrnod gwaith, gan gadarnhau'
```

**Review by danielbe11 — APPROVED** [review]

**Review by velimd — APPROVED** [review]

**Review by nitinprabhu — APPROVED** [review]

**Review by benouaer — APPROVED** [review]

**Pull request merged**